### DISCUSSION TITLE
Do i need to wait for v0.3 for merged PR in PIP?

### DISCUSSION THREAD

**Discussion by talhaanwarch** [autogluon/autogluon#1172] [Q&A] [answered]
Do i need to wait for v0.3 for merged PR recently. 
I mean I am working to use medium_quality_faster_inference preset. But there is an issue #1141. Which is resolved in #1164 . Do I need to wait until you release version 0.3?

**Comment by Innixma** [comment, marked as answer]
To get the latest functionality you can either install the nightly pre-release build or build from source.

To install the pre-release, take a look at the pre-release documentation:

https://auto.gluon.ai/dev/install.html

Generally something along these lines should work:

```
python3 -m pip install -U pip
python3 -m pip install -U setuptools wheel
python3 -m pip install -U "mxnet<2.0.0"
python3 -m pip install --pre autogluon
```

You may have to wait up to 24 hours after a PR is merged before it is available in the pre-release. You can see existing pre-releases here: https://pypi.org/project/autogluon/#history

**Reply by talhaanwarch** [reply]
and how can I install from source?

**Reply by Innixma** [reply]
The instructions are listed in the install page. You have to select `Source`.

You will then see the following instructions:

```
python3 -m pip install -U pip
python3 -m pip install -U setuptools wheel
python3 -m pip install -U "mxnet<2.0.0"
git clone https://github.com/awslabs/autogluon
cd autogluon && ./full_install.sh
```

<img width="817" alt="ag_install_page" src="https://user-images.githubusercontent.com/16392542/121818293-16a54a00-cc3b-11eb-856a-3029ba47676f.png">
